src/main/java/pro/sky/telegrambot/model/NotificationTask.java

```diff
@@ -1,83 +1,21 @@
 package pro.sky.telegrambot.model;
 
+import lombok.Data;
+
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import java.time.LocalDateTime;
 import java.util.Objects;
 
 @Entity
+@Data // lombok - по умолчанию настраивает конструктор, геттеры, сеттеры итд
 public class NotificationTask {
     @Id
     @GeneratedValue
```

Comment:
```suggestion
    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
Лучше указать стратегию